workload/resources/BaseExecutor.js

```diff
@@ -0,0 +1,134 @@
+class BaseExecutor {
```

Comment:
My general question is what's the reason to create the `BaseExecutor` and sub-classes instead of using the existing `BaseRunner` and sub-classes? What't the difference between them?

Reply:
hi @huningxin  Actually this workload integrates some examples(Image Classification/Object Detection/Semantic Segmentation/Skeleton Detection/Emotion Analysis/Facial Landmark Detection), and each sub workload equals relevant example with iterations inferences .
I introduced those **executors**  concept to invoke existed **runners** to do inference , and executor behaves each post processing.

Reply:
Thanks for the explanation. It makes sense to reuse **runners** to do inference. 

In the future, as we discussed, we may consider to merge the functionalities **executors** into examples for better code reusing.